kubevirt-cdi/step2.md

```diff
@@ -1,97 +1,101 @@
-#### Introduction on Containerized Data Importer
+# Use CDI to upload a VM image
 
-[CDI](https://github.com/kubevirt/containerized-data-importer) is a utility designed to import Virtual Machine images for use with Kubevirt.
-
-At a high level, a PersistentVolumeClaim (PVC) is created. A custom controller watches for importer specific claims, and when discovered, starts an import process to create a raw image named *disk.img* with the desired content into the associated PVC.
-
-We will first explore each component and later we will install them. In this exercise we create a hostpath provisioner and storage class. Also, we will deploy the CDI component using the Operator.
-
-#### Install Hostpath Provisioner
-
-Download the hostpath-provisioner deployment YAML and apply it.
-
-`wget https://raw.githubusercontent.com/kubevirt/hostpath-provisioner/main/deploy/kubevirt-hostpath-provisioner.yaml
-kubectl create -f kubevirt-hostpath-provisioner.yaml
-kubectl annotate storageclass kubevirt-hostpath-provisioner storageclass.kubernetes.io/is-default-class=true`{{execute}}
-
-Verify you now have a default storage class. You should see "kubevirt-hostpath-provisioner (default)"
-
-`kubectl get storageclass`{{execute}}
```

Comment:
again, the {{execute}} renders and the remaidner of the page renders incorrectly.